### PR TITLE
Add anonymiser script to anonymise sensitive data in students' code in the test files

### PR DESCRIPTION
**Pull request by Riyas97**

### CODE REVIEW

File: Anonymiser.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+def produceHash(word):
+
+    hash_object = hashlib.sha256(word)
+    hex_dig = " " + hash_object.hexdigest()[:8]
+    return hex_dig
+
+def main():
+
+    path = [Insert path of folder where the test files are stored]
```
Comment (knmnyn) [inline Anonymiser.py:29]: Set a default path and use a better variable name (`default_path`).

File: Anonymiser.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import os
+import re
+import hashlib
+from os.path import join, getsize, isfile
+
```
Comment (knmnyn) [inline Anonymiser.py:5]: This is your library of regex for anonymisation.  You need to mark it more clearly.  Is there any other logic that you are using for this?  If not make that clear that this is the whole thing.  What is the order of precedence in the processing?

**Review by knmnyn — CHANGES_REQUESTED** [review]
Needs work.  

* Needs function and file level documentation.
* Needs an example input and output case.
* Needs a unit test harness to run the example input to verify that an installation achieves the required output.

**Comment by Riyas97** [comment]
Will create a new PR due to new changes and requirements. As such, closing this PR.

**Pull request closed without merging**